### ISSUE TITLE
Add support for customizing Wallaby project cache directory when using WSL2

### ISSUE THREAD

**Issue by kapolos** [wallabyjs/public#2454]
As per the title, since it's already supported on the VS Code plugin.

**Comment by smcenlly** [comment]
When we last reviewed WSL2 support, JetBrains did not have any support for WSL2. With the most recent version of JetBrains editors, you may now open WSL projects using from the WSL Linux file system as `\\wsl$\<distibution>\... `.

Wallaby did not actually require any changes to support WSL2 for JetBrains editors. We've updated [our docs](https://wallabyjs.com/docs/intro/get-started-jetbrains.html#windows-subsystem-for-linux-wsl) to describe that this is now supported.

**Comment by kapolos** [comment]
@smcenlly Thank you for looking into it. I can't seem to get it to work on the latest EAP. What configuration are you using?

Below is my setup and my attempts.

WSL setup:

![image](https://user-images.githubusercontent.com/10794126/85239696-a015a800-b43d-11ea-814b-f751ce08db00.png)

Node in WSL:
```
$ which node
/usr/bin/node
```

### Attempt 1

![image](https://user-images.githubusercontent.com/10794126/85239553-e6b6d280-b43c-11ea-97fe-5fc10dec4c6a.png)
(also tested with Windows style slashes: `\usr\bin\node`)
![image](https://user-images.githubusercontent.com/10794126/85239467-a192a080-b43c-11ea-914f-58b0f586f00d.png)

### Attempt 2


![image](https://user-images.githubusercontent.com/10794126/85239681-8aa07e00-b43d-11ea-8878-3d19f91fdb70.png)


![image](https://user-images.githubusercontent.com/10794126/85239664-78264480-b43d-11ea-9fdb-2312cd5bce10.png)


### Attempt 3

![image](https://user-images.githubusercontent.com/10794126/85239829-5aa5aa80-b43e-11ea-9040-31e46fe4f181.png)


![image](https://user-images.githubusercontent.com/10794126/85239811-48c40780-b43e-11ea-8746-676eb3e0fb1c.png)

**Comment by smcenlly** [comment]
My configuration, system details:

![image](https://user-images.githubusercontent.com/2074089/85240179-80e93b80-b47a-11ea-9f28-7e421b1fc86e.png)

I am using `WebStorm 2020.2 EAP` and `Wallaby for IntelliJ v1.0.177`.

On my WSL instance, I am using `nvm` whose path is set in my `.bashrc` script, so I need to explicitly set the path. This is what my Run Configuration looks like:

![image](https://user-images.githubusercontent.com/2074089/85240278-f5bc7580-b47a-11ea-9c65-66558856d612.png)

**Comment by kapolos** [comment]
Thank you! I'm on the same versions. I just realized there's a different error on the Wallaby console: 

```
Error: EROFS: read-only file system, mkdir '/mnt/c/Users/me/AppData/Local/JetBrains/WebStorm2020.2/wallaby/projects/e48d4dcca875058d-wsl'
    at Object.mkdirSync (fs.js:914:3)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47
```

So, now I know what's going on. For security reasons, I've mounted my C drive inside WSL as read-only but Wallaby needs to write there. Would it possible in a future update to customize the location it uses for writing?

**Comment by smcenlly** [comment]
I'll re-purpose this issue and make it a feature request. Are you blocked from using WSL2 by this?

**Comment by kapolos** [comment]
That's awesome, thank you very much! 🍻 

Not blocked, just can't use Wallaby with my setup and I really enjoy using Wallaby, so it's a bit of a pain not to :)

**Comment by smcenlly** [comment]
We've made some changes that should fix your problem (available in Wallaby core v1.0.918).

We've decided against allowing the folder to be customized and have instead localized Wallaby's cache folder to your WSL instance when using a JetBrains editor with WSL2.

**Comment by kapolos** [comment]
Fantastic!! 🍻 
I'll test and post an update once the new version is out. Thanks again!
